server/game/cards/04-MM/ConsulPrimus.js

```diff
@@ -0,0 +1,29 @@
+const Card = require('../../Card.js');
+
+class ConsulPrimus extends Card {
+    setupCardAbilities(ability) {
+        this.reap({
+            condition: context => context.game.creaturesInPlay.length > 1 && context.game.creaturesInPlay.some(card => card.hasToken('amber')),
```

Comment:
Is this not redundant?

Reply:
Not really, we need at least 2 creatures, one of which must have amber on it.

Reply:
But does the target condition not already cover this?

Reply:
No, it does not.
We had the same discussion by the time when ChantOfHubris was implemented 😄 
I cannot have just 1 creature with amber. Target looks for a creature with amber, but I need to have at least another creature, that's why I added the condition.

Reply:
Then it should just be the first half, as the second half is covered by the target

Reply:
Ok, I will try it

Reply:
I removed the extra condition check